src/FedKeyPairGen/GenesisMiner.cs

```diff
@@ -0,0 +1,65 @@
+﻿using System;
+using NBitcoin;
+
+using Stratis.Bitcoin.Features.PoA;
+
+using Xunit;
+using Xunit.Abstractions;
+
+namespace FedKeyPairGen
+{
+    public class GenesisMiner
+    {
+        private readonly ITestOutputHelper output;
+
+        public GenesisMiner(ITestOutputHelper output)
+        {
+            this.output = output;
+        }
+
+        //[Fact]
```

Comment:
I don't really understand what marking this as a fact does?

Reply:
Ah so that you can run these methods one by one?

Reply:
It means you can run those functions from within visual studio and get the output from the test in the test output. Same as you were doing locally using LinqPad, but this way you don't need to use LinqPad or change Main() to run the functions. In the current version, those functions sit here unused, and someone is expected to change code or copy paste them in LinqPad to run them I think.

Reply:
![image](https://user-images.githubusercontent.com/4638821/48899319-34292300-ee47-11e8-9dd3-687c4aecdd98.png)

Reply:
if someone prefers console or LinqPad really up to them, there is nothing that worked before that doesn't anymore

Reply:
we want to make this a little console app. This way of using tests to run this thing is weird. It should be run once for each network so not something we should run all the time.

Reply:
Maybe it's time for someone to actually create this nice little app so we can package it and be done with it.

Reply:
ok then, let's put this in like this.
As long as this goes before the release and we have a little console app at the end.
I can package it with Warp.

Reply:
cool 👍